docs/api/en/scenes/Scene.html

```diff
@@ -30,6 +30,11 @@ <h3>[property:Boolean autoUpdate]</h3>
 		When it isn't, then you have to maintain all matrices in the scene yourself.
 		</p>
 
+		<h3>[property:Float backgroundBlur]</h3>
+		<p>
+			Sets the blur of the background. Only influences environment maps assigned to [page:Scene.background]. Valid input is a float between *0* and *1*. Default is *0*.
```

Comment:
Sets the blurriness of the background.

Hmm... shininess, roughness, metalness, glossiness.

TBH, I think `blurriness` would be the proper term, and consistent with three.js nomenclature.

Reply:
@WestLangley just to confirm – you're suggesting we change the name of the parameter and not just the word in the sentence here, correct?

@Mugen87 Sorry for adding churn here. I'm OK with either term. 🙏

Reply:
@donmccurdy correct. JMHO. :-)